frontend/src/components/TaskTable.tsx

```diff
@@ -161,10 +161,16 @@ export const TaskTableUnrated: React.FC<{
   useEffect(() => {
     const type = getType(userInfo?.roles, currentRoadmap?.id);
     if (type === RoleType.Admin) {
-      if (userInfo && currentRoadmap && allUsers)
+      if (userInfo && currentRoadmap && allUsers && tasks) {
```

Comment:
The `tasks` is `Task[]`, so the check should be unnecessary, and possible null checks should belong to the user of this component

```suggestion
      if (userInfo && currentRoadmap && allUsers) {
```